aks/provider.tf

```diff
@@ -1,10 +1,15 @@
 provider "azurerm" {
   # whilst the `version` attribute is optional, we recommend pinning to a given version of the Provider
   version = "~> 1.27"
+  
```

Comment:
Shouldn't this be more recent?

Reply:
Yes, but that seemed unrelated, and I didn't want to mix things up anymore than they are already. I'll set versions to match the packaging in a separate PR.